tools/trap/trap.c

```diff
@@ -24,6 +24,9 @@
  ****************************************************************************/
 
 #include "trap.h"
+#include "../../os/include/tinyara/config.h"
```

Comment:
Could you let me know why includes this?

Reply:
It is used for `CONFIG_SMARTFS_MAXNAMLEN`
The maximum possible size of the file path string

Reply:
1. But there is a definition for that at line 29.
[CONFIG_SMARTFS_MAXNAMLEN](https://github.com/Samsung/TizenRT/pull/5738/files#diff-d50c5afb6cd42629b54e5c4ead10ed2e4057df258a2e1f16d7a8f09a326679e9R29)
2. If user does not have built source, then this will be failed. Do we need to build TizenRT to use TRAP?

Reply:
Sorry I need to remove the one in line 29
And I agree with point 2 given by you

Would you suggest I continue with a manually defined dummy config for the purpose of this tool?

Reply:
Thapa also reads some config value on the TRAP. Could you check it with her about how to get a config value? Let's use the same way.

Reply:
I checked tools/trap/ramdumpParser.py
This also uses the TizenRT .config file (Line 1075)

How about adding a comment in HowToUseTrap.md file about the need for building source code for making the config file available?

Reply:
I think 
1. trying to read .config file from default location
2. If no .config there, feedback an error mesage about no .config and wait an input of configuration file 

Many developer uses the tool without source code so that I think getting necessary file separately is better.

Reply:
Pending, will take care of this

Reply:
I pushed a slightly different version.
My estimate is that if the config.h file is not present because the user did not build the code, it is unlikely that they would have the config.h or config file present separately.
Even if they do, I feel it is better to ask the user for the config value directly instead of the file path.
The value is this config is always (so far in all defconfigs) = 32

Also, dynamic inclusion of the provided file path as a header file in C would not be easy.